src/posts/pitfalls-of-accessible-components.md

```diff
@@ -0,0 +1,90 @@
+
+<figure role="figure" aria-label="It's all in the application.">
+	<img alt="A plastic playset is placed on the roof edge of an appartment building." src="/img/posts/pitfalls-of-accessible-components/playset.png" />
+	<figcaption>It's all in the application.</figcaption>
+</figure>
+
+## Page-specific: content
+
+One page is not the other. And it is precisely in the parts that vary that things can of course still go wrong.
+
+### Content
+
+A bit of a tinkering perhaps, but good to be aware of. You write content for every page. It must be accessible. You can think of avoiding complex language, and avoiding descriptions based on specific senses (such as an "open the red link" or "the text to the right of this").
+Content also goes beyond paragraphs and text blocks. Don't forget your alternate texts for images, or captions and transcripts for other media files.
+
+### The page title
+
+Match this as much as possible with the main heading on your page (the `<h1>`) and the names of the links pointing to the page. If there are repeating parts in the title (such as the name of the website), place them as far back in the title as possible. This makes scanning (also with screen readers) easier. The unique and distinctive part comes first.
+In a webshop you get something like: “Product - category - webShopName.nl”
+
+## Page-specific: structure
+
+Perhaps you have something reusable for this, such as templates. That's very nice. However, experience shows that these are not always available and do not have priority.
+
+### Landmarks
+
+You want to have your landmarks in order for each page:
+- Place your navigation in a `<nav>`
+- Place the most important content in a `<main>`
+- Use a `<header>` at the beginning of your page and a `<footer>` at the end
+- Do you have information aside? Then use a `<aside>`
+
+### Heading structure
+
+Your headings help people navigate through your page. Just like you can scan headlines in a newspaper, you want to do the same on a website.
+The `<h1>` is for the main heading; for what your page is about. Below that you want to create a kind of tree structure of your headings. All content logically (both visually and in code) falls under one of your headings. You go from main topic (`<h1>`) to sub-topic (`<h2>`), to sub-sub-topic (`<h3>`), and so on. Never move forward more than one step at a time.
+
+### Focus Order
+
+Make sure that every page you tab through navigates logically. This is easier if you don't use a tab index with a value higher than 0. Creative application of CSS grid or flexbox can also cause the visual order to differ from that in the code.
+Do you want to make it completely easy for your visitor? Add a skip link. That is a (often hidden) link at the beginning of the page that allows you to "jump" to the content.
+
+## Difficult to test
+
```

Comment:
One thing I try to do is not have a heading followed by another heading, as some folk might think there's missing content. 

Can we have a segue sentence or tow to introduce the following content and why you consider it difficult?